medical_physician/data/medical_specialties.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016-2017 LasLabs Inc.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record model="medical.specialty" id="medical_specialty_gp">
```

Comment:
I suggest that we complete based on this list:

https://en.wikipedia.org/wiki/Specialty_(medicine)#Specialties_that_are_common_worldwide